src/components/Sidebar.js

```diff
@@ -0,0 +1,82 @@
+import React, { Fragment } from 'react';
+import { Link } from 'gatsby';
+
+// Toggles sidebar menus, only one may be open at a time
+function toggleMenu(id) {
+  const element = document.getElementById(id);
+
+  const scanners = document.getElementById('Scanners');
+  const persistenceProviders = document.getElementById('Persistence Providers');
+  const hooks = document.getElementById('Hooks');
```

Comment:
This is really not React like using the DOM directly.
Is there a good reason here to "opt" out of react and doing it plain js style?